### DISCUSSION TITLE
Transition guide for ranger users

### DISCUSSION THREAD

**Discussion by grebeard** [gokcehan/lf#1376] [Show and tell]
A few weeks ago I found this wonderful file manager and decided to give it a try. After using ranger for years, it took me some time to configure lf to behave like I was used to.

So I decided to write down all the steps needed to make lf behave more like rangers default to help make the transition easier:
https://github.com/grebeard/lf-howto

Im sure there are still quite a few things I have missed, but I would appreciate any feedback or suggestions to add.

**Comment by gokcehan** [comment]
@grebeard Thank you very much for this work. Just in case you haven't seen before, we have a wiki page dedicated for [ranger](https://github.com/gokcehan/lf/wiki/Ranger). Feel free to add a link to your repo in this wiki page and/or add some content to this page. There has been complaints about this ranger page being quite ineffective, so I think many people would appreciate such additions.

**Reply by grebeard** [reply]
Thanks, I integrated the guide into the wiki page.
